src/main/java/org/twilmes/sql/gremlin/processor/FieldMapVisitor.java

```diff
@@ -95,6 +96,20 @@ private GremlinToEnumerableConverter getConverter(int fieldIndex, String field,
             List<String> fieldNames = join.getRowType().getFieldNames();
             final String chosenField = fieldNames.get(fieldIndex);
             return fieldMap.get(join).get(chosenField);
+        } else if (node instanceof EnumerableCalc) {
```

Comment:
In my case, I always get this object as instance of EnumerableCalc instead of EnumerableJoin. By look deeper into the inputs, I will be able to get the converter out of these fields

Also, When doing 3 tables or more table joins, I see parser gives path that are joining two tables that has no relationship described in SchemaConfig, why? I think it should be problem is Calcite, here I was thinking of providing schema to checking in TraversalVisitor, but was not able to find left and right gremlin table in any of these nodes, so this will be a TODO item to check why it join two tables that has no relationship defined.